client/src/containers/FormBuilderLoader/createIdentifier.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Creates a dot-separated identifier for forms generated
+ * with schemas (e.g. FormBuilderLoader)
+ *
+ * @param props
+ * @returns {string}
+ */
+const createIdentifier = (props) => {
```

Comment:
Yeah, I agree. I had originally named it `getIdentifierFromProps` but that seemed a bit too specific. I really can't imagine this being used anywhere else. The whole purpose of abstracting it was because sometimes you don't have the `this` context, e.g. `mapStateToProps`. It really should be a static member of `FormBuilderLoader`.

How about I move it into the `FormBuilderLoader` file, and rename it to `createFormIdentifierFromProps`?